### PR TITLE
Improvements to the paper

### PR DESCRIPTION
**Pull request by pramitchoudhary**
Updates to section 
1. Decision Boundary
2. Surrogate DT
3. Minor changes and text consistency overall.

### CODE REVIEW

File: tex/multinomial_tex/nips_multinomial_2018.tex [hunk 1]
```diff
@@ -129,43 +130,86 @@ \subsection{Data}
...
+			\label{fig:surrogate_dt}
+		\end{center}
+	\end{subfigure}%
+	\caption{}
 \end{figure}
 
-Prescribed methods for training $h_{\text{tree}}$ do exist \cite{dt_surrogate1} \cite{dt_surrogate2}. In practice, straightforward cross-validation and pruning approaches are often sufficient. Moreover, comparing cross-validated training error to traditional training error can give an indication of the stability of the single decision tree $h_{\text{tree}}$.
-
-Elegantly handles high cardinality targets.
+In this learning task, the target labels are derived from the base estimators learned function. \cite{dt_surrogate1} \cite{dt_surrogate2} provide detained information for training $h_{\text{tree}}$. In practice, straightforward cross-validation and pruning approaches are often sufficient. Moreover, comparing cross-validated training error to traditional training error can give an indication of the stability of the single decision tree $h_{\text{tree}}$. Such form of surrogate models can elegantly handle high cardinality targets maintaining high level of fidelity and faithfulness to the Oracle(base estimator function $g$).
```
Comment (navdeep-G) [inline tex/multinomial_tex/nips_multinomial_2018.tex]: Still unsure about the term `base estimator`. What does it mean in this context? I think its better to refer to `$g$` as it is. Also, `base estimator` makes me think we are doing some type of ensembling as this term is used a lot in that literature. 

Also, not sure about using the term `Oracle` here. Better to leave the sentence as:
`Such form of surrogate models can elegantly handle high cardinality targets maintaining high level of fidelity and faithfulness to the function $g$`

File: tex/multinomial_tex/nips_multinomial_2018.tex [hunk 2]
```diff
@@ -129,43 +130,86 @@ \subsection{Data}
...
 
 \subsection{Decision Boundary Plots}
+Decision Boundaries is an informative interpretation technique to understand the characteristics of multiclass predictive model by visualizing the distances of the data elements to a model's learned decision plane. The boundary helps in providing a direct reference to the complexity of a classifier and the dataset. Figure 2 (a) and (d) illustrates different decision boundaries constructed using a GBM(Gradient Boosted Machine) and a MLP(Mulit-layer Perceptron classifier).
```
Comment (navdeep-G) [inline tex/multinomial_tex/nips_multinomial_2018.tex]: Was a MLP built to get this? If so, is it in a notebook?

File: tex/multinomial_tex/nips_multinomial_2018.tex [hunk 3]
```diff
@@ -115,57 +116,109 @@ \subsection{Spaces}
 
 %\section{Simulated Data Experiments}
 
-\section{Global Analysis}
+\section{Analysis using Global and Local Interpretataion}
 
 \subsection{Data}
-Subsequent sections will use simulated data that empirically demonstrates the desired relationships between input feature importance and interactions in the input space $\mathbf{X}$, the label space $f(\mathbf{X}) = \mathbf{Y}$, a GBM model to be explained $g_{\text{GBM}}$, and a decision tree surrogate $h_{\text{tree}}$. Data with a known signal-generating function depending on four input features with interactions and with eight noise features is simulated such that: 
-
+Subsequent sections will use simulated data to empirically demonstrates the desired relationships and interaction between input feature space $\mathbf{X}$, the output space $f(\mathbf{X}) = \mathbf{Y}$. Simulated data is generated using a pre-defined signal-generating function using four input features with interactions and with eight noisy features with ground truth containing three unique categories(class 1, class 2, class 3) as defined below,
```
[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: @navdeep-G feel free to change the text as you seemed consistent with the literature.

File: tex/multinomial_tex/nips_multinomial_2018.tex [hunk 4]
```diff
@@ -115,57 +116,109 @@ \subsection{Spaces}
 
 %\section{Simulated Data Experiments}
 
-\section{Global Analysis}
+\section{Analysis using Global and Local Interpretataion}
```
[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: i changed it back

[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: Done

File: tex/multinomial_tex/nips_multinomial_2018.tex (hunk 3 above)
[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: @navdeep-G feel free to change the text as you seemed consistent with the literature.

File: tex/multinomial_tex/nips_multinomial_2018.tex [hunk 5]
```diff
@@ -115,57 +116,109 @@ \subsection{Spaces}
...
 
 \subsection{Data}
-Subsequent sections will use simulated data that empirically demonstrates the desired relationships between input feature importance and interactions in the input space $\mathbf{X}$, the label space $f(\mathbf{X}) = \mathbf{Y}$, a GBM model to be explained $g_{\text{GBM}}$, and a decision tree surrogate $h_{\text{tree}}$. Data with a known signal-generating function depending on four input features with interactions and with eight noise features is simulated such that: 
-
+Subsequent sections will use simulated data to empirically demonstrates the desired relationships and interaction between input feature space $\mathbf{X}$, the output space $f(\mathbf{X}) = \mathbf{Y}$. Simulated data is generated using a pre-defined signal-generating function using four input features with interactions and with eight noisy features with ground truth containing three unique categories(class 1, class 2, class 3) as defined below,
 \begin{equation}
```
[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: I may be wrong but something was off with the previous text. I didn't check in the pdf, feel free to compare and change the text as seemed necessary.

File: tex/multinomial_tex/nips_multinomial_2018.tex [hunk 6]
```diff
@@ -186,7 +239,7 @@ \subsection{Shapley Global Feature Importance}
...
...f tree shap. Shapley explanations are estimated on $g_{\text{GBM}}(\mathbf{X})$ for a simulated test set $\mathbf{X}$ with known signal-generating function $f$. Results are presented in Figure \ref{fig:global_feature_imp}. Firstly, the Shapley explanations are shown globally across all class outcomes in a stacked bar chart broken down by absolute global Shapley values per class outcome. This is a good way to see a overall picture of Shapley explanations for multinomial classifiers. Secondly, the Shapley explanations are broken down per class outcome in subsequent charts. All feature contributions for $\text{num}_1, \text{num}_4, \text{num}_8$ and $\text{num}_9$ are seen as most important across all class outcomes both in the global stacked bar chart and per class outcome, which is expected based on Equation \ref{eq:f}. However, they are not seen in the same order. For example, class 0 and class 2 share the same ordering of $\text{num}_1, \text{num}_4, \text{num}_8$ and $\text{num}_9$ but class 1 does not ($\text{num}_1$ and $\text{num}_9$ are swapped). This information can be used to investigate why the Shapley explanations differ between different class outcomes at a global level.
```
[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: I don't think .tex diffs are captured well on git. May be we can use the previous pdf checked in the master and compare the changes. Let me know if i could be of help. But, fastest way would be to 1. accept the changes, 2. refer the pdf and change the text as needed.

[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: I am not sure if git handles .tex diff well. Wouldn't comparing the previous .pdf(master) help to resolve such issues. That might be faster

File: tex/multinomial_tex/nips_multinomial_2018.tex (hunk 5 above)
[the remark this answers is not in the document]
Comment (pramitchoudhary) [reply tex/multinomial_tex/nips_multinomial_2018.tex]: Also, i changed the use of base estimator to original estimator but neutral to any other lingual as needed. Feel free to changes it.

**Comment by pramitchoudhary** [comment]
@navdeep-G @jphall663 merging the changes.

**Pull request merged**